flink-python/pyflink/table/table_config.py

```diff
@@ -44,18 +47,55 @@ def parallelism(self):
     def parallelism(self, parallelism):
         self._parallelism = parallelism
 
+    @property
+    def timezone(self):
+        return self._j_table_config.getTimeZone().getID()
+
+    @timezone.setter
+    def timezone(self, timezone_id):
+        if timezone_id is not None and isinstance(timezone_id, str):
+            j_timezone = self._jvm.java.util.TimeZone.getTimeZone(timezone_id)
+            self._j_table_config.setTimeZone(j_timezone)
+        else:
+            raise Exception("TableConfig.timezone should be a string!")
+
+    @property
+    def null_check(self):
+        return self._j_table_config.getNullCheck()
+
+    @null_check.setter
+    def null_check(self, null_check):
+        if null_check is not None and isinstance(null_check, bool):
+            self._j_table_config.setNullCheck(null_check)
+        else:
+            raise Exception("TableConfig.null_check should be a bool value!")
+
+    @property
+    def max_generated_code_length(self):
+        return self._j_table_config.getMaxGeneratedCodeLength()
```

Comment:
Do we need this method for Python API(this is only using in Java code gen).?

Reply:
I think this method is needed as Python API program is just an API and the Java code-gen logic is shared by both Java/Scala/Python API.